content/en/docs/contribute/style/content-organization.md

```diff
@@ -35,6 +35,11 @@ weight: 10
 For page weights, it can be smart not to use 1, 2, 3 ..., but some other interval, say 10, 20, 30... This allows you to insert pages where you want later.
 {{% /note %}}
 
+{{< note >}}
+Since most documents don't have a date, reusing the same value for multiple pages can make the order of sidebar for localizations different from that of English.
```

Comment:
Can we simplify it like the following ?

```suggestion
When assigning a page weight to a document, the page weight should not be overlapped with other documents within the same directory (section).
```

Reply:
I think it might be better to combine the notes here, since they are related to page weights.  Maybe something like this?

```
For page weights, it can be smart not to use 1, 2, 3 ..., but some other interval, say 10, 20, 30... This allows you to insert pages where you want later. Additionally, each page within the same directory (section) should have a different page weight. This makes sure that content is always organized correctly, especially in localized content. 
```

Reply:
Thanks @seokho-son @a-mccarthy !!
I have accepted the reviews :bow: